modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -810,7 +849,7 @@ object SeqexecEngine {
     SequenceView(seq.id, SequenceMetadata(instrument, obsSeq.observer, obsSeq.seqGen.title), st.status, engineSteps(seq), None)
   }
 
-  private def toSeqexecEvent[F[_]](ev: EventResult[SeqEvent], qState: EngineState[F]): SeqexecEvent = {
+  private def toSeqexecEvent[F[_]](ev: EventResult[SeqEvent], qState: EngineState[F]): Stream[F, SeqexecEvent] = {
```

Comment:
The change to `Stream` for the result allows to produce more than one `SeqexecEvent` in response to a `SeqEvent`.

Reply:
Thanks, I was wondering about that